### PR TITLE
Add storage samples

### PR DESCRIPTION
**Pull request by mcardosos**
Fixes https://github.com/Azure-Samples/azure-sdk-for-go-samples/issues/100

Thanks for your contribution! Please let us know what problems this PR addresses or what functionality it adds. Please link to other related issues and PRs to speed up review. Thank you!

### CODE REVIEW

**Comment by vladbarosan** [comment]
CI seems to be failing...

**Comment by vladbarosan** [comment]
@mcardosos also same thing here, lets rebase now that the fix for CI is in and see if everything is ok

File: storage/account.go [hunk 1]
```diff
@@ -18,6 +19,11 @@ import (
 	"github.com/Azure/go-autorest/autorest/to"
 )
 
+func getAccountName() string {
+	accountName := "azuresamplesgo" + helpers.GetRandomLetterSequence(10)
+	return strings.ToLower(accountName)
```
Comment (vladbarosan) [inline storage/account.go]: i think there is a length limit for storage account name, are we respecting it ?

Comment (mcardosos) [reply storage/account.go]: 3 - 24 characters :)

**Pull request merged**